installer.nsh

```diff
@@ -1,21 +1,68 @@
 !macro customInstall
-  NSISdl::download https://aka.ms/vs/17/release/vc_redist.x64.exe "$INSTDIR\vc_redist.x64.exe"  
+  # Check if Visual C++ Redistributable is already installed
+  # Check for the 64-bit version in the registry
   
-  ${If} ${FileExists} `$INSTDIR\vc_redist.x64.exe`
-    ExecWait '$INSTDIR\vc_redist.x64.exe /passive /norestart' $1
-
-    ${If} $1 != '0' 
-      ${If} $1 != '3010'
-        MessageBox MB_OK|MB_ICONEXCLAMATION 'WARNING: Warcraft Recorder was unable to install the latest Visual C++ Redistributable package from Microsoft.'
-      ${EndIf}
+  Var /GLOBAL VCRedistInstalled
+  StrCpy $VCRedistInstalled "0"
+  
+  # Check for Visual Studio 2015-2022 Redistributable (they use the same key)
+  # This checks for the minimum version required
+  ReadRegStr $0 HKLM "SOFTWARE\Microsoft\VisualStudio\14.0\VC\Runtimes\x64" "Version"
+  ${If} $0 != ""
+    # Version format is like "v14.29.30133.00"
+    # Extract major version number for comparison
+    StrCpy $1 $0 2 1  # Get 2 chars starting from position 1 (skips 'v')
```

Comment:
This seems rather fragile - although it's probably fine. Not a big fan of the 2 char assumption here. It's also just ignoring any non-major version changes. Although I guess that's probably fine.